cloud/deployment/deployment.go

```diff
@@ -447,6 +447,15 @@ func Update(deploymentID, label, ws, description, deploymentName, dagDeploy stri
 		Executor:  currentDeployment.DeploymentSpec.Executor,
 	}
 
+	// change the executor if requested
+	if executor != "" && spec.Executor != executor {
+		// if going from CE -> KE and if multiple queues exist,
+		// only a new default queue will persist
+		printWarning(executor, len(currentDeployment.WorkerQueues))
```

Comment:
We should take a user consent here, so users dont accidentally override the executor?

Reply:
Also, any challanege if they migrate from KE -> CE?

Reply:
@kushalmalani The message is informational to let the users know what the change in the executor means to the underlying worker queues only if the deployment has multiple existing worker queues and the executor is being changed from KE -> CE.

Reply:
What if i want to persist all my worker queues..Can i abort this operation?

Reply:
@kushalmalani no way to abort without a `^C`. Will add consent and canceling ability.